config/swagger.yaml

```diff
@@ -204,6 +204,32 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security:
+  /key-blocks:
+    post:
+      tags:
+        - external
```

Comment:
I see this is external, not internal.

Reply:
Do you prefer `internal`? I think it should be external so the miners can submit the blocks. I'd probably add a new tag `mining` or similar a let the node operator decide if the endpoint should be allowed.